src/main/java/service/read/impl/ReadFileServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package service.read.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import service.read.ReadFileService;
+
+public class ReadFileServiceImpl implements ReadFileService {
+    private static final String CANT_READ_MESSAGE = "Can't read from file: ";
+
+    @Override
+    public List<String> readFromFile(String filePath) {
+        File file = new File(filePath);
+        try {
+            List<String> transactions = Files.readAllLines(file.toPath());
```

Comment:
Better to use Files.readAllLines(Path.of(filePath), than you don't need to create new file.